README.md

```diff
@@ -89,17 +89,27 @@ In order to test caching locally,
 
 To run stress testing against the Salesforce instance, refer to the documentation in the [stress testing folder](load_testing/load_testing.md)
 
-## Pulling and pushing translations from [our project on Phrase](https://app.phrase.com/accounts/city-county-of-san-francisco/projects/dahlia-sf-dahlia-web/dashboard)
+## Translations process with [Phrase](https://app.phrase.com/accounts/city-county-of-san-francisco/projects/dahlia-sf-dahlia-web/dashboard)
 To get started working with our Phrase translations, you will need to:
 
 1. Install the Phrase CLI with `brew tap phrase/brewed && brew install phrase`
-1. [Create an access token for Phrase](https://app.phrase.com/settings/oauth_access_tokens). Save it for future use in e.g. LastPass
+1. [Create an access token for Phrase](https://app.phrase.com/settings/oauth_access_tokens). Save it for future use in Lastpass and as a local env var.
+1. Save the access token as an env var so you don't have to pass it to the phrase commands `export PHRASE_ACCESS_TOKEN="YOUR_ACCESS_TOKEN"`
 
-**To upload new strings or translations to Phrase**
-Run `grunt phrasePull --phraseAccessToken=[your access token]`
+### When strings are added to or deleted from locale-en.json
```

Comment:
Can you split this into the following cases:

When locale-en.json is edited (new keys added only)
When locale-en.json is edited (when some keys are added and some are deleted)
When locale-en.json is edited (when keys are renamed)
When locale-en.json is edited (when keys keep the same name but translation strings change)

Even if some of them have the same processes it would be helpful, since right now I don't know what to do for the bottom two.